Extensions/shuffle+.js

```diff
@@ -368,7 +368,13 @@
 	}
 
 	async function fetchArtistTracks(uri) {
-		const { queryArtistDiscographyAll, queryArtistOverview } = Spicetify.GraphQL.Definitions;
+		const { queryArtistDiscographyAll } = Spicetify.GraphQL.Definitions;
+		const queryArtistOverview = {
+			name: "queryArtistOverview",
+			operation: "query",
+			sha256Hash: "35648a112beb1794e39ab931365f6ae4a8d45e65396d641eeda94e4003d41497",
```

Comment:
Where is this coming from?

Reply:
It's the definition coming from older spotify version which is not present in newer ones

Reply:
Let's leave a comment to document that!